project-management/src/main/java/life/qbic/projectmanagement/application/measurement/validation/MeasurementProteomicsValidator.java

```diff
@@ -112,23 +112,28 @@ public ValidationResult validate(ProteomicsMeasurementMetadata measurementMetada
   public ValidationResult validateRegistration(MeasurementRegistrationInformationPxP metadata,
       ProjectId projectId) {
     var validationPolicy = new ValidationPolicy();
+    var result = ValidationResult.successful();
+
+    for (String sampleId : metadata.measuredSamples()) {
+      result = result.combine(validationPolicy.validationProjectRelation(sampleId, projectId));
+    }
```

Comment:
Heya @KochTobi, Awesome work! 
However this does not solve the issue if the user registers samples from a different experiment within the same project, since we only check on a project level, while samples are registered on an experimentlevel.